### ISSUE TITLE
Hanging and taking insane amounts of memory on loading apkg

### ISSUE THREAD

**Issue by louisrli** [patarapolw/ankisync2#4]
```
$ ps aux | grep python
louis     8416 47.6 66.6 14788800 11071084 pts/1 S  17:34   5:54 python3 main.py
```

I took this straight from the README, nothing else:

```
from ankisync2.apkg import Apkg, db     
apkg = Apkg("nouns.apkg")
```

nouns.apkg is a 5.5mb apkg file with audio media.

basically the program started hanging 
```
$  ps aux | grep python  
louis     8416 47.6 66.6 14788800 11071084
```

The fourth and fifth columns are, unfortunately, [in kilobytes](https://superuser.com/questions/117913/ps-aux-output-meaning)

so that's about 14gb of memory + swap being taken if i'm not mistaken (i left it running for a while to see what would happen before it started freezing up my machine)

anyway -- seems to indicate that my DB (it's a shared deck) is doing something that makes this program apparently very unhappy and looping. It got as far as creating a a 13.6mb collection.anki20, after that the program just seemed to be repeatedly consuming more memory.

happy to attach nouns.apkg if you want to look into it.

**Comment by patarapolw** [comment]
OK, I will. See your `noun.apkg`

**Comment by louisrli** [comment]
[Russian Most Common Nouns.zip](https://github.com/patarapolw/ankisync2/files/6498734/Russian.Most.Common.Nouns.zip)

**Comment by patarapolw** [comment]
Should be fixed in latest version (0.3.2) now. I really made a big mistake of over committing (and not in transaction).
